src/utils/internal.ts

```diff
@@ -676,3 +677,14 @@ export function toHumanReadable<T>(obj: T): HumanReadableType<T> {
 
   return obj as HumanReadableType<T>;
 }
+
+/**
+ * @hidden
+ */
+export function orderTransactionPermissionsValues(
```

Comment:
You shouldn't mutate the original object but rather return a new one. Also, just have it be able to accept null as a value (and return null in that case), and remove the optionize where you're using it

In fact, since it's only being used in once place, I recommend not having it as an util here but rather simply use the logic in place